src/components/AvatarWithImagePicker.tsx

```diff
@@ -260,14 +259,15 @@ function AvatarWithImagePicker({
             {
                 icon: Expensicons.Upload,
                 text: translate('avatarWithImagePicker.uploadPhoto'),
-                onSelected: () => {
-                    if (Browser.isSafari()) {
-                        return;
-                    }
-                    openPicker({
-                        onPicked: showAvatarCropModal,
-                    });
-                },
+                onSelected: () =>
+                    Modal.close(() => {
```

Comment:
Do we have to use `Modal.close` in each menuitem's `onSelected` function? What would happen if we forgot to use `Modal.close` in future `onSelected`?

Reply:
Not in every one but if a menu item opens a new modal/screen it will be needed. It's more of a workaround than an ideal solution 👩🏽‍💻 If we forget to add `Modal.close` the new modal will not open. However I think it won't be hard to find in other usages. What do you think? 🤔

Reply:
I was thinking that maybe some kind of code comment could be useful then 🤔